src/lib/components/ModifyForm.svelte

```diff
@@ -100,7 +100,7 @@
 			cancel();
 		}
 
-		let result = checkNoOverlappingRsvs(Settings, submitted, $reservations);
+		let result = checkNoOverlappingRsvs(Settings, rsv, submitted, $reservations);
```

Comment:
when writing js/ts functions, ideally if arguments are >3 it is typically converted into a config pattern, thus turning it into 1 arg function:
ie
```
checkNoOverlappingRsvs({
   settings: settings,
   reservation: rsv,
   submitted,
   reservations: $reservations,
}):

```

Reply:
this prevents devs from memorizing which arg is on which index.

Reply:
makes sense, will try to do this in the future